rsrc/menus/ScenEditor.rc

```diff
@@ -86,6 +87,10 @@ BEGIN
     END
     POPUP "&Scenario"
     BEGIN
+        MENUITEM "Launch From Here",            IDM_SCEN_LAUNCH_HERE
+        MENUITEM "Launch From Start",           IDM_SCEN_LAUNCH_START
+        MENUITEM "Launch From Town Entrance",   IDM_SCEN_LAUNCH_ENTRANCE
```

Comment:
Hmm… this kind of feels like it belongs in the Town menu, rather than the Scenario menu, maybe? Though I'm not sure…

Reply:
My thinking was, if it goes in the Town menu, it should be called "Launch Scenario From Town Entrance", but that would be a really long menu item name.